README.rst

```diff
@@ -181,24 +181,29 @@ If you do not have **hdf5** or **netcdf4** installed, you can use conda to insta
 
 ::
 
-    conda install hdf5 netcdf4
+    conda install -c conda-forge hdf5
+    conda install -c conda-forge netcdf4
 
 Then install **hddm** via **github**.
 
 :: 
 
-    pip install numpy==1.21
-    pip install --no-cache-dir pymc==2.3.8
-    pip install kabuki
-    pip install hddm==1.0.1
+    pip install "numpy < 1.23.0"
+    pip install "cython <1.0.0"
+    pip install pymc==2.3.8 # backend probabilistic programming framework (DO NOT USE CONDA HERE)
+    # pip install git+https://github.com/hddm-devs/kabuki # (Optional, the pip version as dependency of HDDM should work)
+    pip install hddm # main package
     
-    # Optional (but necessary for LAN extension)
-    pip install torch torchvision torchaudio
+    # Optional
+    pip install torch torchvision torchaudio # The LAN extension makes use of these
 
 To make use of the LAN fuctionalities, you need to install `pytorch`_.
 
 ::
 
+    pip install torch==1.13.1 torchvision==0.14.1 torchaudio==0.13.1
```

Comment:
@AlexanderFengler I reinstated the repeat of the torch install instruction on line 198/204. you should probably remove one of these (I think torch==1.13.1 is outdated relative to your docker file ).